.circleci/kickoff_qa_tests.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# This script kicks off QA smoke or regression tests
+
+GIT_REPOSITORY="https://github.com/astronomer/${TEST_REPO_NAME}.git"
+
+if [[ -z "${TEST_REPO_NAME}" ]]; then
+    echo "The TEST_REPO_NAME environment variable must be defined" >&2
+    exit 1
+fi
+
+read -r -d '' USAGE <<EOF
+kickoff_qa_tests.sh <test_type> <airflow_version>
+
+Arguments:
+  airflow_version (required) - the version of Airflow to test
+  test_type (required)       - the type of tests to kick off, can be either 'smoke' or 'regression'
+EOF
+
+if [[ $# -lt 2 ]]; then
+    echo "$USAGE" >&2
+    echo
+    echo "You must specify both arguments" >&2
+    exit 1
+fi
+
+AIRFLOW_VERSION=$1
+
+TEST_TYPE=$(echo "$2" | tr '[[:upper:]]' '[[:lower:]]')  # smoke or regression
+if [[ "$TEST_TYPE" = "smoke" && "$TEST_TYPE" = "regression" ]]; then
+    echo "The specified test type must be either 'smoke' or 'regression'" >&2
+    exit 1
+fi
+
+# IMAGE_TAG=...
+
+
+
+# Update the repo with the latest
+if [[ -d "${TEST_REPO_NAME}" ]]; then
+    cd "${TEST_REPO_NAME}"
+    git fetch
+    git reset --hard origin/$(git branch --show-current)
+else
+    git clone ${GIT_REPOSITORY}
+    cd ${TEST_REPO_NAME}
+fi
+
+git branch --show-current
```

Comment:
I left this in intentionally so it appears in CircleCI logs. It's one less thing to try to debug without logs.